packages/vscode-extension/src/debug/depsChecker/funcToolChecker.ts

```diff
@@ -131,10 +145,13 @@ export class FuncToolChecker implements IDepsChecker {
       );
     }
 
-    if (!isInstallationValid) {
-      this._telemetry.sendEvent(DepsCheckerEvent.funcValidationError);
+    if (!isVersionSupported || !hasSentinel) {
+      this._telemetry.sendEvent(DepsCheckerEvent.funcValidationError, {
+        "func-v": String(isVersionSupported),
```

Comment:
Is it better to use `"func-v": String(portableFunc)` since it suits better to the name?